config/tizen/iotjs_tizen.patch

```diff
@@ -126,27 +119,41 @@ index b3b51f3..598036a 100644
  )
  set(JERRY_HOST_SNAPSHOT
      ${CMAKE_BINARY_DIR}/${DEPS_HOST_JERRY}/bin/jerry-snapshot)
+diff --git a/config/mbedtls/config-for-iotjs.h b/config/mbedtls/config-for-iotjs.h
+index 6581586..745ebbf 100644
+--- a/config/mbedtls/config-for-iotjs.h
++++ b/config/mbedtls/config-for-iotjs.h
+@@ -57,7 +57,9 @@
+  *
+  * Comment to disable the use of assembly code.
+  */
++#ifndef TIZEN_ASAN_BUILD
+ #define MBEDTLS_HAVE_ASM
++#endif
+ 
+ /**
+  * \def MBEDTLS_HAVE_TIME
 diff --git a/config/tizen/packaging/iotjs.spec b/config/tizen/packaging/iotjs.spec
-index 408c800..9a4f66c 100644
+index 5be21f0..da1fadc 100644
 --- a/config/tizen/packaging/iotjs.spec
 +++ b/config/tizen/packaging/iotjs.spec
 @@ -3,7 +3,7 @@ Version: 1.0.0
  Release: 0
  Summary: Platform for Internet of Things with JavaScript
  Group: Network & Connectivity
 -License: Apache-2.0
-+License: Apache-2.0 and BSD-1-Clause and BSD-3-Clause and MIT and Sun Freely Redistributable Licesne
++License: Apache-2.0 and BSD-1.0 and BSD-3-Clause and MIT
  URL: https://www.iotjs.net/
  Source:     %{name}-%{version}.tar.gz
  Source1:    %{name}.pc.in
-@@ -83,6 +83,7 @@ cp %{SOURCE1001} .
+@@ -86,6 +86,7 @@ V=1 VERBOSE=1 ./tools/build.py \
    --external-include-dir=/usr/lib/glib-2.0/include/ \
    --compile-flag=-D__TIZEN__ \
-   --compile-flag=-fPIC \
+   --compile-flag=-DENABLE_DEBUG_LOG \
```

Comment:
Is this intentional?

Reply:
Current patch file is invalid. So, I changed it to the current iotjs.spec file.